### PR TITLE
Email generation now lists all common interests between mentor and mentee

### PR DESCRIPTION
**Pull request by giorgiovilardo**
To further discussion a bit, let's start with a PR aiming to expand the list of shared interests of the Mentee+Mentor pair in the pairing email.

Tested by creating a copy of the CSV schema file and adding fake data.

### CODE REVIEW

File: MentorMatchmaker/MentorMatchmaker/MatchMakingDomainOperations.fs [hunk 1]
```diff
@@ -102,16 +102,16 @@ let rec createUniqueMentorshipMatches (collectingPairing: CollectingMentorshipPa
         let mentor = potentialMatch.Mentor
         let sessionHours = generateMeetingTimes mentee.MenteeInformation.MentorshipSchedule mentor.MentorInformation.MentorshipSchedule
         if sessionHours.Length > 0 then
-            let confirmedMentoshipMatch = {
+            let confirmedMentorshipMatch = {
```
Comment (giorgiovilardo) [inline MentorMatchmaker/MentorMatchmaker/MatchMakingDomainOperations.fs]: Typo here, fixed

**Comment by giorgiovilardo** [comment]
I'm very new with F#, so I don't know if I made the correct types and whatnot choices; LMK!

**Comment by Kavignon** [comment]
Let me take a look at the code by cloning your branch but a quick glance didn't pick up on anything bad!

Other than allowing people to decide which topic they'd like to focus on, was there any other reason for this addition?

**Comment by Kavignon** [comment]
We're also now conflicting with master and therefore, it should get resolved!

**Comment by giorgiovilardo** [comment]
> Other than allowing people to decide which topic they'd like to focus on, was there any other reason for this addition?

AFAIK, no! There was no explicit request for this feature, but I thought that if we want mentees to choose more than one topic, they should know all the topics they got matched for with their mentor :)

> We're also now conflicting with master and therefore, it should get resolved!

Should just be a matter of reformatting everything

**Comment by giorgiovilardo** [comment]
ok should be good now @Kavignon

**Pull request merged**